### PR TITLE
File base

### PR DESCRIPTION
**Pull request by guaiwu012**
因文件过大，使用网盘分享

### CODE REVIEW

**Comment by why-in-Shanghaitech** [comment]
Hi! Thank you very much for your contribution! I'll review this PR in a few days.
I've noticed that this is your first contribution. I'll add you to the contributor list once the pull request is accepted.

File: courses/ARTS1422/ARTS1422.01_Spring_2024/Lecture Slides 教学课件/98e74de8425145ea9ca48d7a329cdc4.png
Comment (why-in-Shanghaitech) [inline courses/ARTS1422/ARTS1422.01_Spring_2024/Lecture Slides 教学课件/98e74de8425145ea9ca48d7a329cdc4.png:1]: These files do not appear to exceed the 100M size limit. If possible, try not to use third-party tools to upload files.

File: courses/ARTS1422/meta.json [hunk 1]
```diff
@@ -2,6 +2,20 @@
...
+            "course_id": "ARTS1422.01",
+            "department": "信息科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "未知",
+            "evaluation": "Assignments: Coding and  Paper Presentation (20%)  • Quiz (10%)  • The quiz will be only with  a two-sided A4 cheat  sheet  • Top VIS Competition (10%)  • Course Project (60%)",
+            "teacher": [
+                "liquan"
+            ],
+            "ta": []
```
Comment (why-in-Shanghaitech) [inline courses/ARTS1422/meta.json:17]: If there exists a TA, but you do not know who, please add "unknown". Leave this blank means there is no TA in this course.
```suggestion
            "ta": [
                "unknown"
            ]
```

File: courses/CS173/meta.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
+    "status": "activate",
```
Comment (why-in-Shanghaitech) [inline courses/CS173/meta.json:2]: A course is activated iff the instructor uses iTechX as the major platform and now is the course semester.
```suggestion
```

File: courses/CS173/meta.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+            "department": "信息科学与技术学院",
+            "credit": "3",
+            "hours": "48",
+            "prerequisite": "未知",
+            "evaluation": "作业：15% • 涉及知识以及工具的应用以及文献的阅读。 • 随堂：12%测验，3% attendance • 考试：25% • 课程项目：45% • 项目于学期初宣布，以小组为单位完成课题。项目贯穿整个课程期间， 将撰写报告并汇报项目，分为开题、中期以及结题。 • 课程总分5%以内的bonus • 部分（~4%）来自课程项目 • 部分（~1%）来自课堂回答",
+            "teacher": [
+                "zhanghp"
+            ],
+            "ta": [
+                "unknown"
```
Comment (why-in-Shanghaitech) [inline courses/CS173/meta.json:19]: The TA is listed in the 1st slide.

**Review by why-in-Shanghaitech — CHANGES_REQUESTED** [review]
There are several places that are possible to be improved. You may change directly in your personal space, or I could also make the changes for you later.

**Comment by why-in-Shanghaitech** [comment]
Again, thank you for your great contribution! These contents are now available at iTechX.
I have added you to the iTechX contributor list.

**Comment by guaiwu012** [comment]
I didn't check the message in time because I was busy before. Thank you for helping to upgrade the document I uploaded. In addition, itechx is a very good platform, thank you for everything you do for the students in shanghaitech!

**Pull request merged**